images/metering-ansible-operator/roles/meteringconfig/tasks/reconcile_presto.yml

```diff
@@ -49,6 +49,9 @@
       - template_file: templates/presto/presto-serviceaccount.yaml
         apis: [ {kind: serviceaccount} ]
         prune_label_value: presto-serviceaccount
+      - template_file: templates/presto/presto-prometheus-auth.yaml
+        apis: [ {kind: serviceaccount} ]
+        prune_label_value: presto-prometheus-auth
```

Comment:
I think we may need a `create` field if this template file isn't going to be always used.

Reply:
Done in 1a067bf